solutions/monorepo/README.md

```diff
@@ -5,9 +5,9 @@ This is a monorepo example with a single Next.js site ([./app](./app)) that has
 - [./packages/ui](./packages/ui): Exports UI components that use TypeScript and Tailwind CSS
 - [./packages/utils](./packages/utils): Exports utilty functions that use TypeScript
 
-The monorepo is using [Turborepo](https://turborepo.org/) and [yarn workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/) to link packages together, but it can also work with [npm workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces#using-workspaces).
+The monorepo is using [Turborepo](https://turborepo.org/) and [npm workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces#using-workspaces) to link packages together, but it can also work with [yarn workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/).
 
-> In addition to the local packages, it's also using a external package called [`@vercel/examples-ui`](../../packages/ui) that's inside this same repository, it uses `next-transpile-modules` to transpile its UI components that use CSS Modules. `next-transpile-modules` has an impact in build times so when possible, we recommend transpiling packages instead, like shown in the `packages` folder in this example.
+> In addition to the local packages, it's also using a external package called [`@vercel/examples-ui`](../../packages/ui) that's inside this same repository, it uses `next-transpile-modules` to transpile its UI components that use CSS Modules. `next-transpile-modules` has an impact in build times, but that's okay for local packages, if you wish to publish you package you should compile it instead.
```

Comment:
Should we maybe add a general link to publishing? 

https://docs.npmjs.com/packages-and-modules/contributing-packages-to-the-registry

Reply:
There's not much on the npm side that needs to be changed, it's mostly about compiling the package with tsc, SWC, or similar. I intent to include a package in the example that does that but right now it's a bit complicated for this example.